### DISCUSSION TITLE
[how-to] openstreetmap as detail widget

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#16643] [How to]
Hi,

To continue the work started with a geolocation field type #16602, now I need to have the contact map as widget in summary view.

It works as expected, you will find code below if you also need this mod.

````
/custom/layouts/basic/modules/Vtiger/Detail/Widget/DetailViewMap.tpl
/custom/layouts/basic/modules/Vtiger/Detail/Widget/DetailViewMapConfig.tpl
/custom/modules/Settings/Widgets/actions/SaveAjax.php
/custom/modules/Vtiger/widgets/DetailViewMap.php

````
Regards,

[DetailViewMapWidget.zip](https://github.com/YetiForceCompany/YetiForceCRM/files/9973797/DetailViewMapWidget.zip)

**Comment by mariuszkrzaczkowski** [comment]
We will check your code.
One of the clients ordered a new geolocation uitype, we are in the process of adding it to the system. It will be integrated with the map, the working appearance of the field:
![image](https://user-images.githubusercontent.com/10198654/200952548-c3016f96-6e12-4cde-9123-f2240be2ccf6.png)
